service/src/main/java/aero/minova/cas/servicenotifier/ServiceNotifierService.java

```diff
@@ -237,29 +266,23 @@ public void registerProcedureNewsfeed(Table inputTable) {
 	 */
 	public void unregisterProcedureNewsfeed(Table inputTable) {
```

Comment:
Kann es sein, dass diese Methode gar nicht verwendet wird?

Reply:
An sich war mal die Idee, dass Dienste über REST Calls regstiert werden können und dafür wurden die Basis-Methoden in die Services schon mal geschrieben.

Wenn du möchtest, kann ich die Methode aber auch entfernen. ^^